crud/crud.cpp

```diff
@@ -101,9 +103,31 @@ crud::send_response(const database_msg& request, const bzn::storage_result resul
     env.set_sender("placeholder for daemon's uuid"); // TODO
     // TODO: crypto
 
-    session->send_message(std::make_shared<std::string>(env.SerializeAsString()), false);
-}
+    if (session)
+    {
+        session->send_message(std::make_shared<std::string>(env.SerializeAsString()), false);
+    }
+    else
+    {
+        LOG(warning) << "session not set - response not sent via session";
```

Comment:
Please add at least the enum value of the operation that could not be responded to.

Reply:
oh crap... Forgot this.. Fixed